python/lsst/pipe/tasks/loadReferenceCatalog.py

```diff
@@ -43,8 +43,8 @@
 
 class LoadReferenceCatalogConfig(pexConfig.Config):
     """Config for LoadReferenceCatalogTask"""
-    refObjLoader = pexConfig.ConfigurableField(
-        target=LoadIndexedReferenceObjectsTask,
+    refObjLoader = pexConfig.ConfigField(
+        dtype=LoadReferenceObjectsConfig,
         doc="Reference object loader for photometry",
```

Comment:
I think "for photometry" isn't correct here, and I wonder if this config should be marked deprecated?

Reply:
This is a tricky one that I need to discuss (and didn't realize you weren't here in person).  This is used for which color terms to look up, so `ref_dataset_name` is actually used!  Can we demand that the dataset name from the connection info be sent in, or do we need to create a new config now that we've deprecated ref_dataset_name?

Reply:
Can you remove the "for photometry" thing, since it's not just for photometry? Maybe something like "Configuration for the refcat loader"?